server/src/main/java/org/spine3/server/stand/AggregateStateId.java

```diff
@@ -37,19 +51,33 @@
  * @author Alex Tymchenko
  */
 public final class AggregateStateId<I> {
+
+    private static final AggregateStateIdStringifier stringifier =
+            new AggregateStateIdStringifier();
+
+    static {
+        StringifierRegistry.getInstance()
```

Comment:
Can we call `Stringifiers.fromString` without invoking the `static` initialization block?